config/prow_common.mk

```diff
@@ -153,47 +153,11 @@ update-prow-cluster: update-almost-all-cluster-deployments update-all-boskos-dep
 # In emergency, could easily edit this file, deleting all these lines
 confirm-master:
 	@if git diff-index --quiet HEAD; then true; else echo "Git working space is dirty -- will not update server"; false; fi;
-ifneq ("$(shell git branch --show-current)","master")
```

Comment:
`git rev-parse --abbrev-ref HEAD` can be affected by a dirty working space or something
`git branch --show-current` is exactly designed for this purpose

Reply:
It seems the `git` in the prow-tests image does not support `git branch --show-current`.  I'll take a further look on this tomorrow.

Reply:
Then our git at least a year old... https://stackoverflow.com/a/55088865

Reply:
It's already failing in presubmit-build-tests for other PRs, see https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/knative_test-infra/1767/pull-knative-test-infra-build-tests/1234494216499892227#1:build-log.txt%3A655. 

I added a TODO here to use `--show-current` after we update Git in the prow-tests image. Let's do it in followup PRs?

Reply:
k